x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -70,14 +83,28 @@ export const buildShowBuildingBlockFilter = (showBuildingBlockAlerts: boolean):
             disabled: false,
             type: 'exists',
             key: 'signal.rule.building_block_type',
-            value: 'exists',
```

Comment:
so I believe I unintentionally removed this, but looking through the code, it looks like it might actually be redundant. 

For reference, it looks like `type: exists` is important here to build the exists filter, and `value: exists` is used for the UI label. But if you look at filter_label.tsx line 53, an exists filter gets the `exists` label by default.